apps/davi/src/stores/modules/1_5/fetchers/subgraph/useVotingResults/useVotingResults.ts

```diff
@@ -0,0 +1,36 @@
+import { useERC20Info } from 'hooks/Guilds/erc20/useERC20Info';
+import { useHookStoreProvider } from 'stores';
+import { FetcherHooksInterface } from 'stores/types';
+// import { useGuildConfig } from '../useGuildConfig';
+// import { useTotalLocked } from '../useTotalLocked';
+
+type IUseVotingResults = FetcherHooksInterface['useVotingResults'];
+
+export const useVotingResults: IUseVotingResults = (
+  daoId,
+  proposalId,
+  proposal
+) => {
+  // TODO: chequear estos valores
+  const options = [proposal?.totalVotes[0], proposal?.totalVotes[1]];
```

Comment:
Since we're using only `totalVotes` property from the proposal, I think passing that would be more efficient, thus preventing the hook from firing again if other proposal data changes.

Reply:
thanks for noting this, on a previous version i was using more staff from `proposal` so i did this to avoid getting all the proposal again. 
I ended up using only totalVotes so it's correct to just passing this parameter